wiki/Mapping_Techniques/Inspiration_Gallery_Corner/en.md

```diff
@@ -0,0 +1,176 @@
+# Inspiration Gallery Corner
```

Comment:
Some images are just sliders and some are full screens. It doesn't look nice, does it?